components/Cta/CtaNextLink.tsx

```diff
@@ -1,12 +1,14 @@
+'use client';
 import React from 'react';
-import { Link, GatsbyLinkProps } from 'gatsby';
+import Link from 'next/link';
+import { LinkProps } from 'next/link';
```

Comment:
Gatsby Link becomes Next Link of course 😄  I'll clean up these link components further. At least for now they're all linking to the right places.